### PR TITLE
GH-31: Added Empty Frame on App Init

### PR DESCRIPTION
**Pull request by Owen-Griffin**
can also be used as placeholder. Simply an empty frame.

* Issue: gh-31

### CODE REVIEW

File: gui/app.py [hunk 1]
```diff
@@ -40,8 +41,9 @@ def build_app(self, destroy: bool = False) -> None:
         self.alliance_frame = AllianceFrame(self)
         self.settings_frame = SettingsFrame(self)
         self.api_key_frame = APIKeyFrame(self)
+        self.empty_frame = EmptyFrame(self)
         self.sidebar.place(x=0, y=0, height=600, width=150)
-        self.current_frame = self.lookup_frame
+        self.current_frame = self.empty_frame
```
Comment (ItsNeil17) [inline gui/app.py:46]: Initialize emptyframe here. `empty_frame` does not need to be an argument of `App`.

**Review by ItsNeil17 — CHANGES_REQUESTED** [review]

**Comment by Owen-Griffin** [comment]
I have made the requested changes; please review again

**Review by ItsNeil17 — CHANGES_REQUESTED** [review]
LGTM

**Comment by Owen-Griffin** [comment]
I have made the requested changes; please review again

**Review by ItsNeil17 — APPROVED** [review]
LGTM!

**Pull request merged**